enclave/core/sgx/exception.c

```diff
@@ -446,6 +457,17 @@ void oe_virtual_exception_dispatcher(
     }
     else
     {
+        /* The following codes can only be captured with SGX2 and the
+         * MISCSELECT[0] bit is set to 1. */
+        if (is_exit_info_valid &&
```

Comment:
Where do we check `MISCSELECT[0] == 1`?

Reply:
I was curious if this branch should only be activated when `MISCSELECT[0] bit is set to 1`, why that is not in the if-condition of the branch?

Reply:
ah, I misunderstood your question. When `MISCSELECT[0] bit is set to 1`, it is SGX hardware that decides to capture #PF/#GP; i.e., if we configure the enclave with `MISCSELECT[0] bit set to 0`, the #PF/#GP will never be delivered to the enclave. Therefore, we don't need additional software check.